dev/dc.local.yml

```diff
@@ -11,26 +11,6 @@ services:
         environment:
             - PYTHONPATH=/app:/datastore-service:/authlib
             - MYPYPATH=/app:/datastore-service:/authlib
-    datastore-writer:
```

Comment:
IMO the volume mapping in line 9 isn't needed anymore

Reply:
right, removed